docs/vm/live-migration.md

```diff
@@ -62,6 +114,30 @@ When you have [node scheduling rules](./create-windows-vm.md#node-scheduling-tab
 1. Go to the **Virtual Machines** page.
 1. Find the virtual machine in migrating status that you want to abort. Select **⋮ > Abort Migration**.
 
+:::note
+
+- The `Abort Migration` menu is available when the VM already has a running or pending migration process.
+
+- Don't click `Abort Migration` if it is created by the [batch-migrations](#automatically-triggered-batch-migrations)
+
+:::
+
+## Automatically triggered batch-migrations
```

Comment:
I am not sure if this section is necessary. Is this a general FYI for the readers?

Reply:
Seems to me that you want to differentiate between system-initiated vs. user-initiated migration, right? Can you elaborate more on why the action source is important to the user and what the user can do with that info? What user decisions will be affected by this knowledge? E.g., is it the case that we don't want user to terminate system-initiated migration?